docs/howto/account-billing/forgot-your-password.md

```diff
@@ -0,0 +1,55 @@
+# Resetting your password or reclaiming your username
+
+Via the {{gui}}, you can reset a lost password or reclaim your username.
+
+## Reset lost password
+
+Begin by navigating to the https://{{gui_domain}}. At the bottom left-hand side of the page, click on the Forgot your password? button.
```

Comment:
Please replace "https://{{gui_domain}}" with "[{{gui}}](https://{{gui_domain}})". Also, put double quotes around "Forgot your password?". (Those buttons have long labels, so while I wouldn't say I like double quotes, in this guide, I think it makes perfect sense to use them.)